source/docs/advanced/production-ready/media-middleware.md

```diff
@@ -147,18 +147,18 @@ To simplify this process we devised a smart method to determine these numbers in
 11. Using the above information and the different sizes available for your image, you can build a `sizes` value that matches your scenario. Check [example below](#image-sizes-example) for a hands-on exercise.
 
 ```js
-var { getSizesStats, getImage, resetIsMyImage, clearSizesStats } = ((
+let { getSizesStats, getImage, resetIsMyImage, clearSizesStats } = ((
   isMyImage = (img) => {
     return (
       // IMPORTANT UPDATE THE CONDITION BELOW TO MATCH THE IMAGE YOU WANT TO TRACK
       (img.alt || "").trim().toLowerCase() ===
-        "your-image-alt".trim().toLowerCase() &&
+        "your-image-alt".trim().toLowerCase() ||
       (img.src || "")
         .trim()
         .toLowerCase()
-        .indexOf("your-image-src".trim().toLowerCase()) >= 0 &&
+        .indexOf("your-image-src".trim().toLowerCase()) >= 0 ||
       (img?.attributes?.some_custom_prop?.value || "").trim().toLowerCase() ===
-        "your-custom-image-prop-value".trim().toLowerCase() &&
+        "your-custom-image-prop-value".trim().toLowerCase() ||
```

Comment:
This might be incorrect, but it is a little excessive checking, but I guess the correct way would be to comment the unnecessary checks.